traefik/mkdocs-material
# Branded base theme

How to test this theme:

- `npm i`
- `pip3 install -r requirements.txt`
- `npm start`
- `mkdocs serve` (in an other session)
- point your browser to localhost:8000

=> product can be switched changing the product property in `mkdocs.yml`. Possible values are `proxy|mesh|enterprise|pilot`

=> I still need to check with @ldez which Pipy account we should use to publish this package.
